Admin/Admin.php

```diff
@@ -2030,12 +2052,15 @@ public function getFilterFieldDescriptions()
     /**
      * {@inheritdoc}
      */
-    public function addChild(AdminInterface $child)
+    public function addChild(AdminInterface $child, $field = null)
     {
         $this->children[$child->getCode()] = $child;
 
         $child->setBaseCodeRoute($this->getCode().'|'.$child->getCode());
         $child->setParent($this);
```

Comment:
I don't know either and I don't want to fix all this by myself. This need a big rework IMO.